### DISCUSSION TITLE
Toast being shown twice

### DISCUSSION THREAD

**Discussion by Devynko** [usernotnull/tall-toasts#573] [General]
Im having an issue where if I dispatch the same toast twice, it starts showing two toast on one button click for example.

**Comment by Devynko** [comment]
I fixed it by removing this line.
import {Alpine, Livewire} from '../../vendor/livewire/livewire/dist/livewire.esm';

**Reply by marchershey** [reply]
You're probably going to need that.

**Reply by Devynko** [reply]
Why? It's wasn't in my app.js to begin with.  Everything seems to be functioning without it.
